docs/readthedocs/source/doc/Chronos/Howto/how_to_use_benchmark_tool.md

```diff
@@ -0,0 +1,135 @@
+# Use Chronos benchmark tool
+This page demonstrates how to use Chronos benchmark tool to benchmark forecasting performance on platforms.
+
+## Basic Usage
+The benchmark tool is installed automatically when Chronos is installed using conda. To get preliminary knowledge of forecasting performance on platform, run benchmark tool with default options using following command:
+```bash
+benchmark-chronos -l 96 -o 720
+```
+```eval_rst
+.. note::
+    **Required Options**:
+
+     **-l/--lookback** and **-o/--horizon** are required options for Chronos benchmark tool. Use **-l/--lookback** to specify the history time steps while use **-o/--horizon** to specify the output time steps.
+```
+By default, the tool will load `tsinghua_electricity` dataset and train a `TCNForecaster` with input lookback and horizon parameters under `pytroch` framework. As it loads, it prints information about hardware, Nano environment variables and benchmark parameters. When benchmarking is completed, it reports the average throughput during training process. Users may be able to improve forecasting performance by following suggested changes on Nano environment variables.
+
+Besides the default usage, more execution parameters can be set to obtain more benchmark results. Read on to learn more about the configuration options available in Chronos benchmark tool.
+
+## Configuration Options
+The benchmark tool provides various options for configuring execution parameters. Some key configuration options are introduced in this part and a list of all options is given in [**Advanced Options**](https://bigdl.readthedocs.io/en/latest/doc/Chronos/Howto/how_to_use_benchmark_tool.html#advanced-options).
+
```

Comment:
I think you may just link to (#advanced-options)? @Oscilloscope98

Reply:
Yes :) We should always use relative reference instead of direct link if you want to link to a section in the same page/another page from the same document, so that we can make sure the section/page we direct to is in the current version.

Reply:
OK, I will pay attention to using relative reference. Thanks yuwen